doc/user/user/analysis-visualizations/index.md

```diff
@@ -11,42 +11,84 @@ IRIDA Analysis Visualizations
+1. Metadata Template selection
+1. Select all switch
+1. List of checkboxes for all metadata fields associated with samples in the analysis
 
-![Open metadata selection btn](images/toggle_metadata_button.png)
+##### Toggling Individual Fields
 
-Individual metadata field by clicking on the field label in the side panel.
+![Metadata Field Selection](images/metadata-field-selection.gif)
 
-![Toggle metadata field](images/toggle_metadatafield.png)
+You can toggle the visibility of all metadata fields by clicking on the switch to the right of the "Select all". 
 
-After unchecking the checkbox next to 'Province' the metadata column is removed from the image.
+Individual fields can have their visibiltiy toggled based on their checkbox. A single field can be made visible by clicking the "Only" button, which is only visible when hovering over a single metadata field in the dropdown.
 
-![Result of removing 'PFGE-BlnI-pattern'](images/toggle_metadatafield_after.png)
+##### Using Metadata Templates
 
-#### Export to SVG
+In addition to being able to toggle individual metadata fields, a [Sample Metadata Templates](../sample-metadata-templates/) can be selected in the dropdown, which will make only the fields present in that template visible. By default, all fields are displayed.
 
-To export a copy of the visualization into SVG format, simply click on the `Export SVG` button in the upper right.
+#### Export to newick, PNG or SVG format
 
-![Export SVG](images/export_svg.png)
+![Export Menu](images/tree-export-menu.gif)
 
-This svg file can be opened in most external imaging software (except Adobe Illustrator for Mac).
\ No newline at end of file
+To export a copy of the tree in newick, PNG or SVG format, click on the download menu icon at the top right corner of the tree area.  The resulting dropdown will allow you to select the type of export you want.
+
+### Tree Zoom Menu (at bottom right of tree ares)
+
+#### Default Zoom
+
+![Default Zoom](images/default-zoom.gif)
+
+Default zoom allows you to increase or decrease the magnification of the whole tree (similar to zooming on Google Maps).
+
+#### Branch Zoom
+
+![Branch Zoom](images/branch-zoom.gif)
+
+Branch zoom compresses or expands the length of the branches.
+
+#### Step Zoom
+
+![Step Zoom](images/step-zoom.gif)
+
+Step zoom increases or decreases the distance between branches.
+
+### Legend
+
+![Legend Opening](images/expand-legend.gif)
+
+The legend can be displayed by clicking on the Legen button to the right of the tree.  This displays the **visible** metadata fields and the colour of each values for that field (more on this below), for example in the above gif the "Lysteria Monocytogenes" is the only value for the "Organism" field and has the same colour in both locations on the tree.  The legend can expanded by dragging from the edge where the legend meets the tree, and click and draged.
```

Comment:
- Typo in Legend
- Values should just be value
- Add `be` after `The legend can`